Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/tracing/settings/SettingsTracingFragment.kt

```diff
@@ -141,7 +137,7 @@ class SettingsTracingFragment : Fragment(R.layout.fragment_settings_tracing),
             try {
                 if (InternalExposureNotificationClient.asyncIsEnabled()) {
                     InternalExposureNotificationClient.asyncStop()
-                    tracingViewModel.refreshIsTracingEnabled()
+//                    tracingViewModel.refreshIsTracingEnabled()
```

Comment:
If intended, why not delete it as with others.

Reply:
Was a reminder while refactoring, removed :+1: